admin/services/pin-generation.service.js

```diff
@@ -0,0 +1,92 @@
+const moment = require('moment')
+const pupilDataService = require('../services/data-access/pupil.data.service')
+const randomGenerator = require('../lib/random-generator')
+const mongoose = require('mongoose')
+const pinValidator = require('../lib/validator/pin-validator')
+const pupilIdentificationFlagService = require('../services/pupil-identification-flag.service')
+
+const fourPMToday = () => {
```

Comment:
@GeorgeChatzigiannis As discussed in a previous PR, the correct camel case would be fourPmToday

Reply:
@pris54 https://msdn.microsoft.com/en-us/library/141e06ef(v=vs.71).aspx
When using acronyms, use Pascal case or camel case for acronyms more than two characters long. For example, use HtmlButton or htmlButton. However, you should capitalize acronyms that consist of only two characters, such as System.IO instead of System.Io.

Reply:
@GeorgeChatzigiannis This is different from the convention I have been using for quite a while (linked to Wikipedia provided in previous PR). We should come up with an agreement on how to deal with these cases.

Reply:
Most of the links I found are supporting capitalization for two character acronyms. I believe we should keep it this way.

Reply:
@GeorgeChatzigiannis i disagree.  This about readability.  It just looks like PMT is the acronym.  Change it to fourPmToday please.